frontend/simple/assets/sass/theme/kindergarten-professional.scss

```diff
@@ -16,31 +16,28 @@ $primary-bg-a: rgba($primary, 0.2);
 $secondary: #a0d10e;
 $tertiary: #f89201;
 
-$success: saturate($secondary, 10%);
-$warning: lighten(adjust-hue($tertiary, 0%), 10%);
+$success: saturate($secondary, 10%); // TODO/REVIEW - maybe should be exactly the same as $secondary
+$warning: lighten(adjust-hue($tertiary, 0%), 10%); // TODO/REVIEW - maybe should be exactly the same as $tertiary
```

Comment:
Hmm, I don't know. Maybe ask Margarida for her opinion?

Reply:
I think they are very similar, we can merge them for simplicity's sake! @sandrina-p @taoeffect